### PR TITLE
chore: add a script to lint package metadata

### PR DESCRIPTION
**Pull request by spirAde**
This makes it possible to verify that all packages in the monorepo are registered in all required places and have the expected metadata in their package.json

Issue: #4673

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [x] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [x] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [x] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: bin/check-package-metadata.js [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+ */
+'use strict';
+
+const path = require('path');
+const fs = require('fs-extra');
+
+const Project = require('@lerna/project');
+
+const MIT_LICENSE_TEXT = `
+MIT license
```
Comment (spirAde) [inline bin/check-package-metadata.js:20]: On the one hand, I’m not sure about this type of check, on the other hand, the license file will be the same everywhere.

**Comment by spirAde** [comment]
@bajtos You also mentioned that check with the team who owns the apiconnection package from the comment here: [https://github.com/strongloop/loopback-next/issues/4673#issuecomment-629278284](url). Without this, the script will not work, it will crash at the test `CODEOWNERS.md`

**Comment by raymondfeng** [comment]
See https://github.com/strongloop/loopback-next/pull/5449

**Comment by raymondfeng** [comment]
@spirAde Thank you for helping out here. We use conventional commits. The commit message should be amended to be something like `chore: add a script to lint package metadata`. See https://github.com/strongloop/loopback-next/pull/5449.

File: bin/check-package-metadata.js (hunk 1 above)
Comment (raymondfeng) [reply bin/check-package-metadata.js:20]: For our monorepo, we run `lb4 copyright` command once a while to make sure the license headers are up to date.

**Comment by spirAde** [comment]
> @spirAde Thank you for helping out here. We use conventional commits. The commit message should be amended to be something like `chore: add a script to lint package metadata`. See https://loopback.io/doc/en/lb4/submitting_a_pr.html#4-commit-changes-to-the-local-feature-branch.

Oh, sorry. I know about conventional commits. I thought I can use label pointer `feature` from here https://github.com/strongloop/loopback-next/issues/4673 as part of commit message. I'll fix it

File: bin/check-package-metadata.js (hunk 1 above)
Comment (spirAde) [reply bin/check-package-metadata.js:20]: Maybe I can take root folder LICENSE file as source of truth, but in this case loopback needs some way how to support this file in actual state? Or I can simplify check for license file

**Comment by raymondfeng** [comment]
@spirAde Another logistic issue - please run `npm run prettier:fix` to fix formatting issue.

**Comment by spirAde** [comment]
@raymondfeng fixed, but one test failed

File: bin/check-package-metadata.js (hunk 1 above)
Comment (spirAde) [reply bin/check-package-metadata.js:20]: Or another solution: take package `spdx-license-list` which is using in `packages/cli`. Anyway, guys, I need your decision what is the sufficient way from your perspective? @raymondfeng @bajtos

Comment (raymondfeng) [reply bin/check-package-metadata.js:20]: Checking the existence of `LICENSE` file is probably good enough as `lb4 copyright` can deal with the consistency.

**Pull request closed without merging**